src/pages/downloads.tsx

```diff
@@ -11,17 +11,89 @@ import {
 import { DataTable } from '../components/UI';
 
 import {
+  ALL_GETH_COMMITS_URL,
   DEFAULT_BUILD_AMOUNT_TO_SHOW,
   DOWNLOAD_OPENPGP_BUILD_HEADERS,
   DOWNLOAD_OPENPGP_DEVELOPER_HEADERS,
-  GETH_REPO_URL
+  GETH_REPO_URL,
+  LATEST_GETH_RELEASE_URL,
+  LATEST_SOURCES_BASE_URL,
+  LINUX_BINARY_BASE_URL,
+  MACOS_BINARY_BASE_URL,
+  RELEASE_NOTES_BASE_URL,
+  WINDOWS_BINARY_BASE_URL
 } from '../constants';
 
 import { testDownloadData } from '../data/test/download-testdata';
 import { pgpBuildTestData } from '../data/test/pgpbuild-testdata';
 import { pgpDeveloperTestData } from '../data/test/pgpdeveloper-testdata';
 
-const DownloadsPage: NextPage = () => {
+export const getServerSideProps: GetServerSideProps = async () => {
+  // Latest release version number
+  const versionNumber = await fetch(LATEST_GETH_RELEASE_URL)
```

Comment:
```suggestion
  const { versionNumber, releaseName } = await fetch(LATEST_GETH_RELEASE_URL)
    .then(response => response.json())
    .then(release => {
      return {
        versionNumber: release.tag_name,
        releaseName: release.name
      }
    });
```

This could be condensed to one call

Reply:
Oh nice! Updated ⚡️